compiler/passes/checkParsed.cpp

```diff
@@ -314,6 +317,21 @@ checkFunction(FnSymbol* fn) {
   }
 }
 
+static void nestedName(ModuleSymbol* mod) {
+  if (mod->defPoint == NULL) {
+    return;
+  }
+
+  ModuleSymbol* parent = mod->defPoint->getModule();
+  if (parent != mod &&
```

Comment:
I thought I remembered the rootModule being its own parent for some strange reason, but I didn't verify that came up.  I'll do that now